controllers/users.controller.js

```diff
@@ -151,5 +161,82 @@ export default {
         return res.status(500).send('Something went wrong')
       }
     }
+  },
+  getOrdersForMasterByUserId: async (req, res) => {
```

Comment:
It is not about the user
it should be like
masters/:id/orders
Or
orders/masterId
As you wish